plugins/Morpheus/stylesheets/base/icons.css

```diff
@@ -436,3 +436,12 @@
 .icon-marketplace:before {
   content: "\e90c";
 }
+.icon-plugin:before {
+  content: "\e90d";
+}
+.icon-add1:before {
+  content: "\e62e";
+}
+.icon-remove:before {
+  content: "\e643";
+}
```

Comment:
Where are these used? Can't find the usage in this PR.

Reply:
@bx80 has added those icons to the font in #21077, but they weren't contained in the css file, causing them to be displayed incorrectly on the ui demo, which no one seems to have noticed before. Guess they are meant to be used somewhere at some point 🤷